app/javascript/follow-button.js

```diff
@@ -0,0 +1,42 @@
+$(document).ready(function() {
+  $('.btn-follow').each(function() {
+    const self = $(this);
+    const following = self.data('follows');
+    const styleClass = following ? 'btn-unfollow' : 'btn-follow';
```

Comment:
```suggestion
    const styleClass = following == 'true' ? 'btn-unfollow' : 'btn-follow';
```
The string _false_ is truthy in js